src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -1293,6 +1302,10 @@ public void setPrelimaryNotification(boolean prelimaryNotification) {
         this.prelimaryNotification = prelimaryNotification;
     }
 
+    public boolean isInteractive() {
+        return isUsingCustomValidator() && customValidatorSettings.isUseInteractiveValidatorInterface();
```

Comment:
It seems to me that this is perhaps an inappropriate overloading of functionality.  The concept `problem is interactive` is (subtly, but IMHO clearly) different from `custom validator is using interactive validator interface`.  It seems like there should be a `Checkbox` somewhere in the problem definition that says "This problem is interactive", and the state of that checkbox should be what is used to determine the return value for method `isInteractive()`.

See also my comment in `EditProblemPane.getCustomValidatorOptionsSubPanel()` regarding the creation of the "Custom Validator 'Checkbox'".

Reply:
See my comments in the comment about the reasoning for a radio group.